src/aws/osml/model_runner/database/ddb_helper.py

```diff
@@ -87,13 +90,17 @@ def get_ddb_item(self, ddb_item: DDBItem) -> Dict[str, Any]:
         """
         return self.table.get_item(Key=self.get_keys(ddb_item=ddb_item))["Item"]
 
-    def put_ddb_item(self, ddb_item: DDBItem, condition_expression: str = None) -> Dict[str, Any]:
+    def put_ddb_item(
+        self, ddb_item: DDBItem, condition_expression: str = None, max_retries: int = 30, max_delay: float = 8
```

Comment:
Are the new max_retries/max_delay parameters used any more on put_ddb_item? Seems like they might be left over from an earlier iteration.

Reply:
Will remove